platform/linuxbsd/export/export.cpp

```diff
@@ -40,8 +40,8 @@ void register_linuxbsd_exporter_types() {
 void register_linuxbsd_exporter() {
 	Ref<EditorExportPlatformLinuxBSD> platform;
 	platform.instantiate();
-	platform->set_name("Linux/X11");
-	platform->set_os_name("Linux");
+	platform->set_name("Linux/BSD");
+	platform->set_os_name("LinuxBSD");
```

Comment:
What was the reasoning behind having both `name` and `os_name`?

Reply:
`set_name()` sets the name displayed in the list of platforms. `set_os_name()` sets rather a technical name (I did not find it displayed somewhere in the interface). Also, a feature tag is formed from `os_name` (converted to lower case).

Reply:
The `os_name` was changed in https://github.com/godotengine/godot/pull/61352. So it looks like we wanted to go the opposite direction and split Linux and BSD? cc @Faless

Reply:
Yeah, Linux and BSD are not compatible systems.
Binaries for Linux do not work on BSD, and vice versa, they have different OS APIs, they are not the same platform even if they share a good amount of source code.
And we must be able to detect which platform we are in if we want gdextensions to work  (and not end up trying to load a linux shared library on BSD).
As I mentioned in the other PR, we might be able to simply add another exporter dedicated to BSD.